jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -101,11 +101,11 @@ def buildExclusions = [
   [ /sharedlibs_shared/,              anyType,     lt(9)   ],
 
   // OSX ---------------------------------------------------
-  [ /^osx1010/,                       anyType,     gte(11) ],
-  [ /^osx1011/,                       releaseType, lt(11)  ],
-  [ /^osx1011/,                       releaseType, gte(13) ],
+  [ /^osx1010/,                       testType,    gte(11) ],
+  [ /^osx1010/,                       releaseType, allVer  ],
   [ /^osx1011/,                       testType,    gte(14) ],
-  [ /^osx1015/,                       releaseType, lt(13)  ],
+  [ /^osx1011/,                       releaseType, allVer  ],
+  [ /^osx1015/,                       releaseType, noVer   ],
```

Comment:
```suggestion
```

Confusingly does nothing, I suggest deleting.

Reply:
Joao decided to use this pattern in the Windows section to be explicit about what's doing what. I figure that including these brings us closer to that explicit state that we might achieve by flipping this from exclude to include; `noVer` are essentially inline documentation so there's no confusion for future editors.
Do you actually find this confusing re osx, or just re how the script works?

Reply:
Confusion is relative, I find it more confusing than

> // Releases for all OS X versions are done on OS 10.15 with approriate target levels, to support notarization

Reading code that superficially might have an effect, but on consideration, does not tends to make me ask myself "did the author intend to write code that does nothing?", so there's that. If you find it a helpful reminder, though, go for it.

Reply:
No, that's fine, have replaced it with a comment. It's inconsistent anyway since it's `releaseType` and really it's `anyType noVer`.